easybuild/easyconfigs/b/Boost/Boost-1.59.0-GCC-5.2.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Boost'
+version = '1.59.0'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'GCC', 'version': '5.2.0'}
```

Comment:
We don't use the GCC toolchain anymore directly. Use `GNU` which is a combination of GCC with binutils. There reason for this is that RHEL 6 has a binutils that is too old to support Haswell hardware.